modules/@apostrophecms/i18n/localizations/en.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Apostrophe core's localizations are all here for ease of maintenance, but *any* module can have localizations and specify what namespace they wind up in.